lib/api/prop-types.js

```diff
@@ -2,12 +2,21 @@ import _ from "lodash";
 import { check } from "meteor/check";
 import * as Schemas from "/lib/collections/schemas";
 
+/**
+ * @file **PropTypes** - React Component PropType methods for validating Tags
+ *
+ * @namespace PropTypes
+ */
+
 const TagSchema = Schemas.Tag.newContext();
 
 export const PropTypes = {};
 
 /**
- * React Component propType validator for a single Tag
+ * @name Tag
+ * @summary React Component propType validator for a single Tag
```

Comment:
Should `propType` be `PropType` to match the `@file` description above?

Reply:
yes! fixed!

Reply:
In line 17 here it's still showing as `propType` to me